.editorconfig

```diff
@@ -0,0 +1,22 @@
+
+# https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = 0
+trim_trailing_whitespace = false
+
+[*.ejs*]
```

Comment:
I think that this is not needed

Reply:
We should remove it from template repo also, as it has same editorconfig file